src/components/Views/Trips.jsx

```diff
@@ -55,15 +38,19 @@ export const Trips = () => {
     return []
   }, [timeseriesQueryResult])
 
-  if (!isLoadingAccountQuery && accounts.length === 0) {
+  if (isLoadingTimeseriesQuery) {
+    return (
+      <Spinner size="xxlarge" className="u-flex u-flex-justify-center u-mt-1" />
```

Comment:
nit: is it possible to quickly test that Spinner presence when isLoadingTimeseriesQuery is true?